spec/system/referrals/user_adds_personal_details_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "User adds personal details to a referral" do
```

Comment:
Proper minor, but could be:

```suggestion
RSpec.feature "Personal details" do
```